snap/snapcraft.yaml

```diff
@@ -124,10 +116,13 @@ parts:
 
       CPU_COUNT=$(lscpu --parse=CPU | grep -v "^#" | wc -l)
 
+      export DESTDIR="$(readlink --canonicalize ${SNAPCRAFT_PART_INSTALL})"
```

Comment:
About the `CPU_COUNT` variable above, Snapcraft should provide `SNAPCRAFT_PARALLEL_BUILD_COUNT`.

Reply:
Ah. I was looking for a variable like that but couldn't find it in the documentation. I'll fix this.